### ISSUE TITLE
False positive IDE0001 when using extern alias

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/roslyn#74681]
**Version Used**: 17.11 Preview 1.1

**Steps to Reproduce**:

1. Have a package reference like `<PackageReference Include="MyPackage" Aliases="MyAlias" Version="Version" />`
2. Have a project reference to a project that contains a type with the same fully qualified name as some type existing in MyPackage
3. Have code like:

    ```csharp
    extern alias MyAlias;
    using PackageType = MyAlias.Fully.Qualified.Name.To.Type;
    ```

4. IDE0001 Suggests `using PackageType = Fully.Qualified.Name.To.Type`

**Diagnostic Id**:

IDE0001

**Expected Behavior**:

No diagnostic

**Actual Behavior**:

false positive diagnostic

**Comment by CyrusNajmabadi** [comment]
Closing out as we do not invest in extern alias scenarios (outside of fixing crashes).
